docs/src/main/sphinx/release/release-387.md

```diff
@@ -0,0 +1,89 @@
+# Release 387 (22 June 2022)
+
+## General
+
+* Update minimum required Java version to 11.0.15. ({issue}`12841`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## Accumulo connector
+
+## Atop connector
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Add support for `data_size` clause when analyzing a table. ({issue}`12814`)
+
+## Druid connector
+
+## Elasticsearch connector
+
+## Google Sheets connector
+
+## Hive connector
+
+## Iceberg connector
+
+* Prevent modifying old snapshots. ({issue}`12860`)
```

Comment:
Can we change to `Fail queries that attempt to modify old snapshots`?

cc: @findepi